### DISCUSSION TITLE
Borg detailed backup report to file

### DISCUSSION THREAD

**Discussion by ueismann** [nextcloud/all-in-one#4486] [Questions] [answered]
Borg generates a detailed output after finishing, which is available in the NC AOI logs as long as the backup container is running. Is there a good way to write this output to a log file in my backup folder (where the backups are stored)?

**Comment by szaimen** [comment, marked as answer]
Hi, you can write the logs to a file e.g. like so:
`docker logs nextcloud-aio-borgbackup > /mnt/backup/borg/backup.logs`

**Reply by ueismann** [reply]
works like a charm, thanks for that
